setup.py

```diff
@@ -166,6 +166,7 @@ def get_exts_for(name):
             "enum34; python_version<'3.4'",
             "funcsigs>=1.0.0; python_version=='2.7'",
             "typing; python_version<'3.5'",
+            "packaging>=20.0",
```

Comment:
do we actually need this? we get this as part of `pkg_resources`, but  better to be explicit?

Reply:
Yes, you do need it in theory. So keep it. Not sure about the version number though.

Reply:
I went semi-arbitrary. I didn't do too much digger, `20.0` was from 2020, seems like API for version parsing is pretty stable going ways back, we could updated this to `>=18.0` which would be 2018+... (now that I think about it... is the major version based on year... `20.0` was in 2020...)